seekinternational/cpp-helpers
# [CPP-2882] Refactor commonjs into es2015 imports/exports.

- Set eslint rule `import/no-commonjs` to `warning`.

What does this PR do?
----------------------

Refactor commonjs into es2015 imports/exports.

How can I test this?
----------------------

`yarn run test` should pass. Additionally, `yarn run lint` should not give warnings on `import/no-commonjs` rules.
